### PR TITLE
FEATURE: SD-1001 adding types for the save as new dashboard post message

### PR DESCRIPTION
**Pull request by vannguyengd**

### CODE REVIEW

**Comment by yenkins** [comment]
Currently destined for [release R209](https://confluence.intgdc.com/display/Dev/R209).
This release is **zero downtime** release.
Code drop: 2020-07-16 00:00.

**Comment by yenkins** [comment]
Story [SD-1001](http://jira.intgdc.com/browse/SD-1001) Post event command for save as new Kpi dashboards
	Status: Backlog
	Priority: Major
	Assignee: Van Nguyen

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/124/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/209/ : FAILURE in 1m 00s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/125/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/211/ : FAILURE in 41s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/126/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/212/ : SUCCESS in 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/130/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/216/ : SUCCESS in 42s

**Comment by ptquang86** [comment]
LGTM

**Comment by dlcd2203** [comment]
LGTM

File: src/embedding/kd.ts [hunk 1]
```diff
@@ -253,6 +258,29 @@ export namespace EmbeddedKpiDashboard {
         return getEventType(obj) === GdcKdCommandType.Save;
     }
 
+    /**
+     * Creates a new dashboard from an existing dashboard
+     *
+     * Contract:
+     *
+     * -  if KD saves as new an existing dashboard, the DashboardSaved event will be posted
+     * -  if KD doesn't have an existing dashboard, CommandFailed is posted
```
Comment (lanhq) [inline src/embedding/kd.ts:267]: should describe more detail this contract for these case
- duplicate in view mode
- duplicate in edit mode
- duplicate locked dashboard

**Comment by yenkins** [comment]
Currently destined for [release R210](https://confluence.intgdc.com/display/Dev/R210).
This release is **zero downtime** release.
Code drop: 2020-07-30 00:00.

**Comment by yenkins** [comment]
Story [ONE-4414](http://jira.intgdc.com/browse/ONE-4414) Manual column resizing in KD
	Status: Completed
	Priority: Major
	Assignee: Ivan Nejezchleb

**Comment by yenkins** [comment]
Story [SD-1001](http://jira.intgdc.com/browse/SD-1001) Post event command for save as new Kpi dashboards
	Status: Hacking
	Priority: Major
	Assignee: Van Nguyen

**Comment by yenkins** [comment]
## client-libs/gooddata-typings-pull-request-compliance-check check failed
### require-active-sprint
- ONE-4414

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/137/ : UNSTABLE in 8s
- gooddata-typings-unit-tests-zuul-docker-el7 : SKIPPED

File: src/embedding/kd.ts (hunk 1 above)
Comment (vannguyengd) [reply src/embedding/kd.ts]: Updated!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/138/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/224/ : SUCCESS in 54s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/139/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/225/ : SUCCESS in 23s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/141/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/227/ : SUCCESS in 55s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/44/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/228/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-typings-fossa-scanning-zuul-el7/8/ : SUCCESS in 12s (non-voting)

**Pull request merged**